projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.html

```diff
@@ -10,4 +10,24 @@
     <ccf-labeled-slide-toggle class="toggle side" [labels]="['L', 'R']"></ccf-labeled-slide-toggle>
     <ccf-labeled-slide-toggle class="toggle gender" [labels]="['Male', 'Female']"></ccf-labeled-slide-toggle>
   </div>
-</ng-template>
+  <hr>
+  <mat-expansion-panel (opened)="load = true" [class.load]="load" class="extraction-menu">
```

Comment:
This `load = true` business is a really BAD solution. A 1 second look at the api of mat-expantion-panel (https://material.angular.io/components/expansion/api#MatExpansionPanel) shows that it has a `expanded` input that you should set to `false`